packages/design-system/src/components/Review/Review.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import ReviewLink from './ReviewLink';
 import classNames from 'classnames';
 import { t } from '../i18n';
 
-export type ReviewHeadingLevel = '1' | '2' | '3' | '4' | '5';
+export type ReviewHeadingLevel = '1' | '2' | '3' | '4' | '5' | '6';
```

Comment:
I added an option for the Review to support `<h6>` tags. Our Stories reflect this as an option and it seemed odd to exclude it as an optional level. I don't think this breaks the API as it's just adding another option instead of taking anything away.